server/channels/app/plugin_api_test.go

```diff
@@ -163,6 +163,42 @@ func TestPublicFilesPathConfiguration(t *testing.T) {
 	assert.Equal(t, publicFilesPath, publicFilesFolderInTest)
 }
 
+func TestPluginAPIGetUserPreference(t *testing.T) {
```

Comment:
Just to be thorough, may be nice to test (and therefore define) what happens if you try to get a non existing preference or from a non existing user.

Reply:
Makes sense. I'll add a couple cases for that.

These tests feel so small compared to my usual test suites, so I can definitely flesh them out :D